start

```diff
@@ -562,7 +562,7 @@ popd() {
 
 function service_status() {
   stellar_core_status &
-  horizon_status
+  horizon_status &
```

Comment:
I think running horizon in background makes sense for when soroban rpc is enabled since it still provides status but doesn't hold up the soroban rpc readiness notification, but otherwise we should probably keep the same behavior of running horizon_status in foreground/blocking to retain same behavior for non soroban rpc usage which represent use cases of quickstart mainly for horizon?

Reply:
I have updated so that it only runs in the background when soroban-rpc is enabled.

PTAL